ansible_base/models/organization.py

```diff
@@ -0,0 +1,30 @@
+"""Organization models."""
+from django.conf import settings
+from django.db import models
+
+from .common import UniqueNamedCommonModel
+
+
+class AbstractOrganization(UniqueNamedCommonModel):
+    """An abstract base class for organizations."""
+
+    class Meta:
+        abstract = True
+
+    description = models.TextField(
+        null=False,
+        default="",
+        help_text="The organization description.",
+    )
+
+    users = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="organizations",
+        help_text="The list of users in this organization.",
+    )
+
+    teams = models.ManyToManyField(
+        settings.ROLE_TEAM_MODEL,
+        related_name="organizations",
+        help_text="The list of teams in this organization.",
+    )
```

Comment:
Can you add a doc in the docs folder talking about this AbstractModel and how to implement it? I'm noting here that applications using DAB will need to implement ROLE_TEAM_MODEL, what happens if they don't implement that? Do they get an error in the migrations?

Reply:
It will fail at import, because `settings.ROLE_TEAM_MODEL` is not defined.

Reply:
We might want to rename that setting. Just about all settings in the RBAC stuff I was working on could stand to get better names.

I also don't think it will matter unless someone actually inherits from the abstract model. The generated migration, IIRC, tends to put in the hard-coded value, although you can use the Django swappable_dependency thing, I have not seen it do anything helpful.